### PR TITLE
feature/connect-sonarcloud

### PR DESCRIPTION
**Pull request by Gh0stF1ghter**

### CODE REVIEW

**Review by MaliyOna — APPROVED** [review]

**Comment by pavelGagarin0870** [comment]
Test coverage missed

File: .github/workflows/sonarcloud.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+jobs:
+  Analysis:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Analyze with SonarCloud
+
+        # You can pin the exact commit or the version.
+        # uses: SonarSource/sonarcloud-github-action@v2.2.0
+        uses: SonarSource/sonarcloud-github-action@4006f663ecaf1f8093e8e4abb9227f6041f52216
```
Comment (5Ka-me) [inline .github/workflows/sonarcloud.yml:50]: Why there is the ID of an action?

**Review by 5Ka-me — CHANGES_REQUESTED** [review]
Remove unnecessary comments. Looks like you steal the code from ChatGPT without adjusting 😶‍🌫️
Add code coverage

File: .github/workflows/sonarcloud.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    - name: Build
+      run: dotnet build --no-restore --configuration Release
+    - name: Test
+      run: dotnet test --no-build --verbosity normal --configuration Release
+    - name: Sonar Scan
+      uses: highbyte/sonarscan-dotnet@v2.3.0
+      with:
+        sonarProjectKey: Gh0stF1ghter_Project-Rent-n-Ride
+        sonarProjectName: Project-Rent-n-Ride
+        sonarOrganization: gh0stf1ghter
```
Comment (5Ka-me) [inline .github/workflows/sonarcloud.yml:31]: Get these values from GH secrets/variables

**Review by 5Ka-me — CHANGES_REQUESTED** [review]

**Review by 5Ka-me — APPROVED** [review]

**Pull request merged**